leptonai/photon/dockerfiles/photon-rocm-runner.Dockerfile

```diff
@@ -1,10 +1,6 @@
 ARG BASE_IMAGE
 FROM ${BASE_IMAGE}
 
-RUN pip install -U pip setuptools wheel
```

Comment:
turns out this is already in install_python.sh https://github.com/leptonai/leptonai/blob/main/leptonai/photon/dockerfiles/install_python.sh#L25